apps/omg_watcher/lib/application.ex

```diff
@@ -109,4 +109,10 @@ defmodule OMG.Watcher.Application do
         :child_chain_exit
     end
   end
+
+  defp deposit_events_callback(deposits) do
+    :ok = OMG.API.State.deposit(deposits)
+    OMG.Watcher.DB.EthEventDB.insert_deposits(deposits)
```

Comment:
Why don't we match that with :ok?

Reply:
I've added `:ok` match on each particular insert - so it's either all pass or fail on first error